arraycontext/impl/jax/__init__.py

```diff
@@ -70,40 +86,43 @@ def zeros(self, shape, dtype):
         import jax.numpy as jnp
         return jnp.zeros(shape=shape, dtype=dtype)
 
-    def from_numpy(self, array: Union[np.ndarray, _ScalarLike]):
-        import jax
-        return jax.device_put(array)
+    def empty_like(self, ary):
```

Comment:
Could you add a few container-using tests for these newly container-allowing functions?

Reply:
Added in 691c879.